bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py

```diff
@@ -59,6 +63,23 @@ def __init__(self, name, bitrate, rate, profile, id_=None, description=None, cus
         self.crf = crf
         self.minKeyframeInterval = min_keyframe_interval
         self.maxKeyframeInterval = max_keyframe_interval
+        self.sampleAspectRatioNumerator = sample_aspect_ratio_numerator
+        self.sampleAspectRatioDenominator = sample_aspect_ratio_denominator
+        self.sceneCutThreshold = scene_cut_threshold
+        self.colorConfig = color_config
+
+    @property
+    def colorConfig(self):
+        return self._colorConfig
+
+    @colorConfig.setter
+    def colorConfig(self, new_color_config):
+        if new_color_config is None:
+            self._colorConfig = None
+        elif isinstance(new_color_config, ColorConfig):
+            self._colorConfig = new_color_config
```

Comment:
elsewhere we're always setting the .value of the enum here?

Reply:
What enum do you mean? colorConfig is an Object